webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -25,27 +25,26 @@ const getNotifications = url => (dispatch) => {
 
   if (isDocumentVisible) {
     API.get(url)
-      .done(onGetNotificationsSuccess)
-      .fail(onGetNotificationsFailed)
-      .always(triggerPolling);
-  } else {
-    // document is not visible, keep polling without api call
-    triggerPolling();
```

Comment:
you skipped all of this functionality on purpose ? (skipping if document is not visible and setting up polling?)